LiteDB/Document/Expression/Methods/DataTypes.cs

```diff
@@ -131,7 +131,11 @@ public static BsonValue DOUBLE(BsonValue value, BsonValue culture)
             }
             else if(value.IsString && culture.IsString)
             {
+#if HAVE_GET_CULTURE_INFO
                 var c = CultureInfo.GetCultureInfo(culture.AsString); // en-US
+#else
+                var c = new CultureInfo(culture.AsString); // en-US
```

Comment:
CultureInfo.GetCultureInfo caches culture, and I don't want add any performance regressions.
So I've added #if here to support both cases.